manifests/local-dind/kind-cluster-v1.12.8.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+k8s_version=v1.12.8
+clusterName=$1
```

Comment:
I guess in most cases there is no need to specify a name. How about giving it a default name, e.g. `${1:-tidb-operator}`